test/src/main/java/io/strimzi/test/k8s/KubeClient.java

```diff
@@ -917,7 +918,7 @@ public String clusterKubernetesVersion() {
      * @return cluster operator pod name
      */
     public String getClusterOperatorPodName(final String namespaceName) {
-        LabelSelector selector = kubeClient(namespaceName).getDeploymentSelectors(namespaceName, "strimzi-cluster-operator");
+        LabelSelector selector = new LabelSelectorBuilder().withMatchLabels(Map.of("strimzi.io/kind", "cluster-operator")).build();
```

Comment:
Maybe use the constant from API module?

Reply:
Transitive dependency

Reply:
Ah yes, sorry, I didn't realize that it's the `test` module